### DISCUSSION TITLE
[Feature Request] Ability to add list item above or below in vim mode

### DISCUSSION THREAD

**Discussion by sinewave** [vslinko/obsidian-outliner#242] [Archive]
Hi, amazing plugin... honestly couldn't use Obsidian without it 🙌 .

Any chance support could be added for "o" and "O" vim commands which add a new line item above or below the current line. 

The advantage of these commands is that they can be triggered from anywhere on the line.

**Comment by vslinko** [comment]
Duplicate of https://github.com/vslinko/obsidian-outliner/discussions/194

**Reply by louiskirsch** [reply]
I am also desperately in need for this!
